VAMobile/src/constants/analytics.ts

```diff
@@ -572,11 +572,20 @@ export const Events = {
       },
     }
   },
-  vama_login_start: (isSIS = false): Event => {
+  vama_login_readme: (isSIS = false): Event => {
+    return {
+      name: 'vama_login_readme',
+      params: {
+        sis: isSIS.toString(),
+      },
+    }
+  },
+  vama_login_start: (isSIS = false, isBiometric = false): Event => {
```

Comment:
same thing here. we haven't used the old login version in years and is no longer supported and can be hardcoded to true or dropped all together I believe

Reply:
I defaulted sis here to true

Reply:
I'd prefer dropping the `isSIS` param altogether to avoid confusion since it's not relevant anymore, unless there's a reason we still want to keep it?